presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -13,37 +13,34 @@
  */
 package io.prestosql.tests.hive;
 
-import io.prestosql.tempto.Requirement;
-import io.prestosql.tempto.RequirementsProvider;
+import io.prestosql.tempto.Requires;
 import io.prestosql.tempto.assertions.QueryAssert;
-import io.prestosql.tempto.configuration.Configuration;
+import io.prestosql.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements.ImmutableNationTable;
+import io.prestosql.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements.ImmutableOrdersTable;
 import io.prestosql.tempto.query.QueryResult;
 import org.testng.annotations.Test;
 
 import java.math.BigDecimal;
+import java.sql.Date;
+import java.time.LocalDate;
 import java.util.function.Consumer;
 
 import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
 import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
-import static io.prestosql.tempto.fulfillment.table.TableRequirements.immutableTable;
-import static io.prestosql.tempto.fulfillment.table.hive.tpch.TpchTableDefinitions.NATION;
 import static io.prestosql.tempto.query.QueryExecutor.query;
 import static io.prestosql.tests.TestGroups.HIVE_VIEWS;
 import static io.prestosql.tests.utils.QueryExecutors.onHive;
 import static io.prestosql.tests.utils.QueryExecutors.onPresto;
 import static java.lang.String.format;
 import static org.testng.Assert.assertEquals;
 
+@Requires({
+        ImmutableNationTable.class,
+        ImmutableOrdersTable.class,
+})
 public class TestHiveViews
         extends HiveProductTest
-        implements RequirementsProvider
 {
-    @Override
-    public Requirement getRequirements(Configuration configuration)
-    {
-        return immutableTable(NATION);
```

Comment:
Why do you remove it? You are using nation table so it should be somehow provisioned. I guess in your case it was provisioned for other test.

Reply:
it's provided with a class-level annotation now